components/esp_hw_support/port/esp32c61/cpu_region_protect.c

```diff
@@ -153,7 +153,11 @@ void esp_cpu_configure_region_protection(void)
     PMP_ENTRY_SET(10, SOC_RTC_IRAM_LOW, NONE);
 #if CONFIG_ULP_COPROC_RESERVE_MEM
     // First part of LP mem is reserved for coprocessor
+#if CONFIG_ESP_SYSTEM_PMP_LP_CORE_RESERVE_MEM_EXECUTABLE
```

Comment:
Can drop this since C61 does not have an ulp

Reply:
Unused ESP32-C61 code has been removed, thanks!